src/engine/specification.md

```diff
@@ -312,6 +354,30 @@ The payload build process is specified as follows:
 
 8. If any of the above fails due to errors unrelated to the normal processing flow of the method, client software **MUST** respond with an error object.
 
+### engine_forkchoiceUpdatedV2
```

Comment:
Are `engine_forkchoiceUpdatedV1` and `engine_forkchoiceUpdatedV2` interchangeable when `payloadAttributes` is `null`?

Otherwise, will there be any sort of restriction to never use `V1` again (or error out) once a certain condition is met?

Reply:
they would work the same way

I think this ties into the versioning question up above

Reply:
generally with the CL APIs we do deprecate stuff after some time, and we could do a similar thing here.

esp given that we will deploy this in a hard fork we can make the old stuff invalid on the new fork